src/blueapi/worker/reworker.py

```diff
@@ -83,8 +87,26 @@ def __init__(
         self._stopping = Event()
         self._stopped = Event()
 
-    def submit_task(self, name: str, task: Task) -> None:
-        active_task = ActiveTask(name, task)
+    def clear_task(self, task_id: str) -> None:
+        del self._pending_tasks[task_id]
+
+    def get_pending_tasks(self) -> List[Task]:
```

Comment:
Should this returns the tasks or the IDs? What is this method intended to be used for?

Reply:
Intended use is the REST API so it should return the tasks

Reply:
Realised we would also need the task ID for the rest API so I promoted `TrackableTask` (formerly `ActiveTask`) to the API level